### PR TITLE
Implemented RobotDestroyed-Event

### PR DESCRIPTION
**Pull request by baldhat**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ microservice:
   map:
     address: ${MAP_SERVICE}
 topics:
-  robot-movement: movement
+  robot-movement: movem
```
Comment (DennisWae) [inline src/main/resources/application.yml:29]: Why did you change this? There are no changes to this topic's name in the specs.

**Review by DennisWae — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (baldhat) [reply src/main/resources/application.yml:29]: This was not intended

**Review by DennisWae — APPROVED** [review]
